mock-hpos-api/authUtils.js

```diff
@@ -1,32 +1,33 @@
 const stringify = require('fast-json-stable-stringify')
 const sha512 = require('js-sha512')
-const { isEmpty } = require('lodash')
 
 // there's some duplication between this file and src/utils/keyManagement.js
 
-const signPayload = (keypair, method, request, body) => {
-  let bodyHash
+const verifySignedRequest = (givenSignature, method, pathIncludingQuery, body, keypair) => {
+  // BUG: real server (hp-admin-crypto) does not properly check the body when
```

Comment:
Would be good to note the version of hp-admin-crypto this refers to.

Reply:
Addressed indirectly by linking to a specific github issue which includes version information